test-framework/core/src/main/java/org/keycloak/testframework/database/DatabaseConfigRepresentation.java

```diff
@@ -0,0 +1,31 @@
+package org.keycloak.testframework.database;
+
+public final class DatabaseConfigRepresentation {
+    private String initScript;
+    private String database;
+    private boolean preventReuse;
+
+    public String getInitScript() {
+        return initScript;
+    }
+
+    public void setInitScript(String initScript) {
+        this.initScript = initScript;
+    }
+
+    public String getDatabase() {
+        return database;
+    }
+
+    public void setDatabase(String database) {
+        this.database = database;
+    }
+
+    public boolean doesPreventReuse() {
```

Comment:
Would just stick with standard get/set conventions here (`does` sounds strange as well ;)):

```suggestion
    public boolean isPreventReuse() {
```